docs/src/main/asciidoc/container-image.adoc

```diff
@@ -88,15 +89,16 @@ For customizing the extension the following properties are available:
 
 .Container Image
 |====
-| Property                                | Type                            | Description                       | Default Value
-| quarkus.container-image.group           | String                          | The group/repository of the image | The ${user.name}
-| quarkus.container-image.name            | String                          | The name of the image             | The application name
-| quarkus.container-image.tag             | String                          | The tag of the image              | The application version
-| quarkus.container-image.registry        | String                          | The registry to use for pushing   | docker.io
-| quarkus.container-image.username        | String                          | The registry username             |
-| quarkus.container-image.password        | String                          | The registry password             |
-| quarkus.container-image.insecure        | Boolean                         | Flag to allow insecure registries | false
-| quarkus.container-image.execution       | Enumeration (NONE, BUILD, PUSH) | The kind of execution (see below) | NONE
+| Property                                | Type                            | Description                                                            | Default Value
+| quarkus.container-image.group           | String                          | The group/repository of the image                                      | The ${user.name}
+| quarkus.container-image.name            | String                          | The name of the image                                                  | The application name
+| quarkus.container-image.tag             | String                          | The tag of the image                                                   | The application version
+| quarkus.container-image.registry        | String                          | The registry to use for pushing                                        |
+| quarkus.container-image.username        | String                          | The registry username                                                  |
+| quarkus.container-image.password        | String                          | The registry password                                                  |
+| quarkus.container-image.insecure        | Boolean                         | Flag to allow insecure registries                                      | false
+| quarkus.container-image.build           | Boolean                         | Flag to enable building of a container image                           | false
+| quarkus.container-image.push            | Boolean                         | Flag to enable pushing of a container image to the configured registry | false
```

Comment:
Question: which image format does it support?

Reply:
Docker 2.2. Jib also provides the option to use the OCI format, should I expose this option via a property?

Reply:
I actually tried the OCI thing, but at this point it's more trouble than it's worth since I couldn't push such an image to docker or quay